src/dex-ui/pages/dao/CreateADAO/forms/NFTDAOGovernanceForm.tsx

```diff
@@ -1,260 +1,46 @@
-              type: "number",
-              placeholder: "Enter Max Supply",
-              register: {
-                ...register("governance.nft.maxSupply", {
-                  required: { value: true, message: "Max supply is required." },
-                  validate: (value) => value >= 0 || "Invalid max supply.",
-                }),
-              },
-            }}
-            isInvalid={Boolean(errors.governance?.nft?.maxSupply)}
-            errorMessage={errors.governance?.nft?.maxSupply && errors.governance?.nft?.maxSupply?.message}
-          />
-          <FormInput<"governance.nft.tokenWalletAddress">
-            inputProps={{
-              id: "governance.nft.tokenWalletAddress",
-              label: "Token wallet address",
-              type: "text",
-              placeholder: "Enter wallet address",
-              register: {
-                ...register("governance.nft.tokenWalletAddress", {
-                  required: { value: true, message: "Token wallet address is required." },
-                  validate: (value) =>
-                    checkForValidAccountId(value) || "Invalid address, please, enter a different one.",
-                }),
-              },
-            }}
-            isInvalid={Boolean(errors.governance?.nft?.tokenWalletAddress)}
-            errorMessage={
-              errors.governance?.nft?.tokenWalletAddress && errors.governance?.nft?.tokenWalletAddress?.message
-            }
-          />
-          <Flex gap="4" justifyContent="flex-end">
-            <Button key="create-token" onClick={createNewToken}>
-              Create New NFT
-            </Button>
-          </Flex>
-        </DAOFormContainer>
-        <Text textStyle="p large regular">Initial token distribution</Text>
-        <DAOFormContainer>
-          <FormInput<"governance.nft.treasuryWalletAccountId">
-            inputProps={{
-              id: "governance.nft.treasuryWalletAccountId",
-              label: "Treasury wallet account id",
-              type: "text",
-              placeholder: "Enter wallet account id",
-              register: {
-                ...register("governance.nft.treasuryWalletAccountId", {
-                  required: { value: true, message: "A treasury account id is required." },
-                  validate: (value) =>
-                    checkForValidAccountId(value) || "Invalid address, please, enter a different one.",
-                }),
-              },
-            }}
-            isInvalid={Boolean(errors.governance?.nft?.treasuryWalletAccountId)}
-            errorMessage={
-              errors.governance?.nft?.treasuryWalletAccountId && errors.governance?.nft?.treasuryWalletAccountId.message
-            }
-          />
+      <Tabs isFitted onChange={handleTabChange} index={tabIndex} variant="unstyled" width="100%">
+        <DAOFormContainer rest={{ padding: "0.1rem" }}>
+          <TabList>
+            <Tab
+              color={Color.Neutral._500}
```

Comment:
Add a style variant to the `TabsStyles.ts` file and use that variant for these tabs. An example should already be in that file (ex. "dao-dashboard-tab" variant).

Reply:
Made the suggested changes.